docs/user/intro/docusaurus.rst

```diff
@@ -0,0 +1,97 @@
+
+Docusarus
+=========
+
+.. meta::
+   :description lang=en: Hosting Docusaurus sites on Read the Docs.
+
+`Docusaurus`_ is a static-site generator that builds a single-page application with fast client-side navigation and out-of-the-box documentation features.
+
+Minimal configuration required to build a Docusaurus project on Read the Docs looks like this,
+specifying a nodejs toolchain on Ubuntu, using multiple :ref:`build <config-file/v2:build>` commands to install the requirements,
+build the site, and copy the output to $READTHEDOCS_OUTPUT:
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+
+    version: 2
+    build:
+    os: "ubuntu-22.04"
+    tools:
+        nodejs: "18"
+    commands:
+        # "docs/" was created following the Docusaurus tutorial:
+        # npx create-docusaurus@latest docs classic
+        # but you can just use your existing Docusaurus site
+        #
+        # Install Docusaurus dependencies
+        - cd docs/ && npm install
+        # Build the site
+        - cd docs/ && npm run build
+        # Copy generated files into Read the Docs directory
+        - mkdir --parents $READTHEDOCS_OUTPUT/html/
+        - cp --recursive docs/build/* $READTHEDOCS_OUTPUT/html/
+
+.. _Docusaurus: https://docusaurus.io/
+
+Supported Features
+------------------
+
+.. csv-table:: Supported Features
+   :header: "Feature", "Description", "Supported"
+
+   "Pull request previews", "Preview changes to your documentation before merging.", "✅"
+   "Versioning", "Supports multiple versions of your documentation.", "✅"
+   "Flyout menu", "Provides a flyout menu for navigation.", "✅"
+   "Search", "Provides full-text search capabilities.", "❌"
+   "Offline formats", "Generates PDF and ePub formats.", "❌"
```

Comment:
If they run a command that ouputs a PDF under `READTHEDOCS_OUTPUT/pdf` this should work, shouldn't it?

Reply:
We don't do anything special for PDF or EPUB for Sphinx or MkDocs either -- I mean, we run a default command for them; but not running that command it doesn't mean we don't support EPUB or PDF on Docusaurus or any other doctool where we don't run this default command.

If they output a PDF or EPUB under that directory, it works in the same way than for MkDocs or Sphinx. I don't think we should communicate these limitations since you can get the exact same output with any doctool.